2019/5xxx/CVE-2019-5615.json

```diff
@@ -1,18 +1,121 @@
 {
-    "CVE_data_meta": {
-        "ASSIGNER": "cve@mitre.org",
-        "ID": "CVE-2019-5615",
-        "STATE": "RESERVED"
-    },
-    "data_format": "MITRE",
-    "data_type": "CVE",
-    "data_version": "4.0",
-    "description": {
-        "description_data": [
+   "CVE_data_meta": {
+      "ASSIGNER": "cve@rapid7.com",
+      "DATE_PUBLIC": "2019-01-30T14:00:00.000Z",
+      "ID": "CVE-2019-5615",
+      "STATE": "PUBLIC",
+      "TITLE": "Rapid7 InsightVM Stored Credential Exposure"
+   },
+   "affects": {
+      "vendor": {
+         "vendor_data": [
             {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+               "product": {
+                  "product_data": [
+                     {
+                        "product_name": "InsightVM",
+                        "version": {
+                           "version_data": [
+                              {
+                                 "version_affected": "<=",
+                                 "version_name": "6.5.49",
+                                 "version_value": "6.5.49"
+                              },
+                              {
+                                 "version_affected": ">=",
+                                 "version_name": "6.5.11",
+                                 "version_value": "6.5.11"
+                              },
+                              {
+                                 "version_affected": "!>=",
```

Comment:
The other two version specifications already explicitly limit this vuln to 6.5.11 - 6.5.49, so lines 29-33 can be dropped/superfluous.

Reply:
word